client/app/index.html

```diff
@@ -33,77 +33,70 @@
-    <script src="/bower_components/angular-resource/angular-resource.min.js"></script>
-    <script src="/bower_components/angular-sanitize/angular-sanitize.js"></script>
-    <script src="/bower_components/angular-timeago/src/timeAgo.js"></script>
+    <!-- build:js /angular.js -->
+    <script src="/lib/angular/angular.min.js"></script>
+    <script src="/lib/angular-route/angular-route.min.js"></script>
+    <script src="/lib/angular-i18n/angular-locale_en-us.js"></script>
+    <script src="/lib/angulartics/dist/angulartics.min.js"></script>
+    <script src="/lib/angulartics/dist/angulartics-ga.min.js"></script>
+    <script src="/lib/angular-animate/angular-animate.min.js"></script>
+    <script src="/lib/angular-bootstrap-npm/dist/angular-bootstrap-tpls.js"></script>
+    <script src="/lib/ui-select/dist/select.js"></script>
+    <script src="/lib/angular-bootstrap-datetimepicker/src/js/datetimepicker.js"></script>
+    <script src="/lib/angular-cookie/angular-cookie.min.js"></script>
+    <script src="/lib/ng-file-upload/dist/angular-file-upload-shim.min.js"></script>
+    <script src="/lib/ng-file-upload/dist/angular-file-upload.min.js"></script>
+    <script src="/lib/angular-growl-v2/build/angular-growl.min.js"></script>
+    <script src="/lib/angular-resource/angular-resource.min.js"></script>
+    <script src="/lib/angular-sanitize/angular-sanitize.js"></script>
+    <script src="/lib/angular-timeago/src/timeAgo.js"></script>
 
-    <script src="/bower_components/angular-ui-router/release/angular-ui-router.min.js"></script>
-    <script src="/bower_components/ui-router-extras/release/modular/ct-ui-router-extras.core.js"></script>
-    <script src="/bower_components/ui-router-extras/release/modular/ct-ui-router-extras.future.js"></script>
-    <script src="/bower_components/angular-translate/angular-translate.min.js"></script>
-    <script src="/bower_components/angular-translate-loader-static-files/angular-translate-loader-static-files.min.js"></script>
-    <script src="/bower_components/messageformat/messageformat.js"></script>
-    <script src="/bower_components/messageformat/locale/en.js"></script>
-    <script src="/bower_components/messageformat/locale/es.js"></script>
-    <script src="/bower_components/angular-translate-interpolation-messageformat/angular-translate-interpolation-messageformat.min.js"></script>
+    <script src="/lib/angular-ui-router/release/angular-ui-router.min.js"></script>
+    <script src="/lib/ui-router-extras/release/modular/ct-ui-router-extras.core.js"></script>
+    <script src="/lib/ui-router-extras/release/modular/ct-ui-router-extras.future.js"></script>
+    <script src="/lib/angular-translate/dist/angular-translate.min.js"></script>
+    <script src="/lib/angular-translate-loader-static-files/angular-translate-loader-static-files.min.js"></script>
+    <script src="/vendor/messageformat/messageformat.js"></script>
```

Comment:
It should be there, GitHub won't let me look at all the files in the interface because there are too many (lol), but I have it locally. Do you not see it when you switch to the repo?